mgmt/api/CoreAPI.cc

```diff
@@ -880,6 +880,21 @@ EventSignalCbUnregister(const char *event_name, TSEventSignalFunc func)
   return cb_table_unregister(local_event_callbacks, event_name, func);
 }
 
+/*-------------------------------------------------------------------------
+ * HostStatusSet
+ *-------------------------------------------------------------------------
+ * Iterates through the RecordsConfig table, and for all stats
+ * (type PROCESS, NODE, CLUSTER), sets them back to their default value
+ * If one stat fails to be set correctly, then continues onto next one,
+ * but will return TS_ERR_FAIL. Only returns TS_ERR_OKAY if all
+ * stats are set back to defaults successfully.
```

Comment:
This explanation raises more questions than it answers. As a consumer of this API, what is it's intended effect? I can't really tell how the explanation of the stats behavior is relevant to the status of some specific host.